runtime/src/telemetry/metrics/status.rs

```diff
@@ -5,6 +5,32 @@ use prometheus_client::{
     metrics::{counter::Counter as DefaultCounter, family::Family},
 };
 
+/// Metric label that indicates whether something is true or false.
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, EncodeLabelValue)]
+pub enum Bool {
+    False,
+    True,
+}
+
+impl From<bool> for Bool {
+    fn from(value: bool) -> Self {
+        if value {
+            Bool::True
+        } else {
+            Bool::False
+        }
+    }
+}
```

Comment:
I was kind of hoping `Status` could be used for this (and originally had a `From<bool>` implemented for status that would map to `Success/Failure`). I wonder if either is better:
- Just using `Status` would satisfy the use-case
- Using a more specific name than `Bool` (and possibly not having it here but instead in the crate/file that actually uses this as a LabelValue

Reply:
I was going to use it but changed course. Happy to just remove it until it is actually useful (and we can cross that bridge then).

Reply:
removed